components/raftstore/src/store/async_io/write.rs

```diff
@@ -880,39 +904,124 @@ where
         cfg: &Arc<VersionTrack<Config>>,
     ) -> Result<()> {
         let pool_size = cfg.value().store_io_pool_size;
-        for i in 0..pool_size {
-            let tag = format!("store-writer-{}", i);
-            let (tx, rx) = bounded(cfg.value().store_io_notify_capacity);
-            let mut worker = Worker::new(
+        self.increase_by(
+            pool_size,
+            StoreWritersMeta {
```

Comment:
Why not keep a copy here and save the trouble of passing it to `resize` everytime.

Reply:
Two points here:
1. Just for less modifications on existing code-style.
If we keep a copy in `StoreWriter`, the specification of `StoreWriter<EK, ER>` should be expanded to `StoreWriter<EK, ER, T: Transport + 'static, N: PersistedNotifier>`. And this will bring a consequential change on objects which include `StoreWriter` as their members, such as `RaftBatchSystem<EK: KvEngine, ER: RaftEngine>` and `Worker<...>`. Separating these parameters for staring a thread as an extra obj, named as `StoreWritersMeta`, will wrap all parameters, and this will introduce less modifications on other existing code-styles.
2. As the `resize` is an OP command for users, it's not called by users frequently. So, just keeping a copy and passing it when `resize` won't introduce much more costs on building and starting threads.